File: OpenAlprWebhookProcessor/angularapp/src/app/settings/openalpr-agent/agentStatus.ts
export class AgentStatus {
    hostname: string;
    isConnected: boolean;
    version: string;
    cpuCores: number;
    cpuUsagePercent: number;
    daemonUptimeSeconds: number;
    diskFreeBytes: number;
    systemUptimeSeconds: number;
    agentEpochMs: number;
    alprdActive: boolean;

    constructor(init?:Partial<AgentStatus>) {
        Object.assign(this, init);
    }
}

File: OpenAlprWebhookProcessor/Settings/AgentStatus.cs
namespace OpenAlprWebhookProcessor.Settings
{
    public class AgentStatus
    {
        public bool IsConnected { get; set; }

        public string Hostname { get; set; }

        public string Version { get; set; }

        public int CpuCores { get; set; }

        public decimal CpuUsagePercent { get; set; }

        public long DaemonUptimeSeconds { get; set; }

        public long DiskFreeBytes { get; set; }

        public long SystemUptimeSeconds { get; set; }

        public long AgentEpochMs { get; set; }

        public bool AlprdActive { get; set; }
    }
}
